dockerfiles/steps/step-epub.bash

```diff
@@ -6,7 +6,13 @@ set -Eeuxo pipefail
 # Ensure $IO_EPUB is empty
 [[ -d $IO_EPUB ]] && rm -rf ${IO_EPUB:?}/*
 
-node --unhandled-rejections=strict "$JS_UTILS_STUFF_ROOT/bin/bakery-helper" epub ./ "$IO_EPUB/"
+extra_vars=""
+(( JS_DEBUG==1 )) && {
+    echo "Javascript debug mode is on"
+    extra_vars+="--inspect-brk=0.0.0.0:9229"
```

Comment:
Other pars use `DEBUG_HOST` and `##_DEBUG_PORT`. Any reason this is different?

Also, it seems like this is missing from other node steps like mathify

Reply:
Will update this part.